AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/config/sofar/sources_id_metadata.json

```diff
@@ -612,10 +612,10 @@
     "latitude_nominal": -34.08,
```

Comment:
Change to the value in the metadata: -34.1

Reply:
Just FYI, the data currently in the API is at 34.08. I don't know if they plan on changing it or if it just a rounding issue, but it will flag the buoy at the wrong coordinates once I implement the error/warning system (still trying to find a good compromise with those...)

Reply:
Changed back to -34.1 (with a warning that it will probably lead to wrong coordinates errors when I implement the message system)